src/main/java/com/arpnetworking/http/Routes.java

```diff
@@ -80,16 +80,19 @@
      * @param metricsFactory Instance of <code>MetricsFactory</code>.
      * @param healthCheckPath The path for the health check.
      * @param statusPath The path for the status.
+     * @param supplementalRoutes Optional supplemental routes.
      */
     public Routes(
             final ActorSystem actorSystem,
             final MetricsFactory metricsFactory,
             final String healthCheckPath,
-            final String statusPath) {
+            final String statusPath,
+            final Optional<SupplementalRoutes> supplementalRoutes) {
```

Comment:
I'm not sure I see the point in providing it at this level. This provides an entry point to hook into the http router, if the customizer wants to federate the routes, they can do so with whatever design pattern (e.g. configuration, injection, scannotation, etc.) that makes sense in their customization. 

Providing multiple supplements could be useful if we or a second-party were to add a supplement and then a third-party wanted to add another. I think that might be overcomplicating this a bit.